taskcluster/docker/inference/Dockerfile

```diff
@@ -27,6 +27,14 @@ RUN apt-get update -qq \
                           libtool \
     && apt-get clean
 
+RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.40.1/install.sh | bash && \
+    export NVM_DIR="$HOME/.nvm" && \
+    [ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh" && \
+    nvm install node && \
```

Comment:
Best specify a version instead of "the current stable version at the time the docker image is built", to avoid breakage at random times when the image is rebuilt (and to ensure the taskcluster cache key is different when the node version changes).

Reply:
Sure. I'll lock it in to a version.